container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -1067,14 +1067,15 @@ instance_groups:
   scripts:
   - scripts/set_vcap_limits.sh
   - scripts/forward_logfiles.sh
-  - scripts/patches/cc_clock_wait_for_api_ready.sh
   - scripts/patches/cc_clock_disable_credhub.sh
   - scripts/patches/fix_monit_rsyslog.sh
   post_config_scripts:
   - scripts/bpm_kube_dns.rb
   jobs:
   - name: global-properties # needs to be first so images use it for processing monit templates
     release: scf-helper
+  - name: wait-for-api # https://trello.com/c/lCGRAGcy/340-migration-errors-with-api-0-crash-during-install
```

Comment:
I missed this comment in the first review run. I don't think this belongs here. Could you expand the comment, instead of referencing a Trello card? (It won't work forever).

Reply:
Will do.

Reply:
Updated locally.